pkg/pipeline/transform/kubernetes/kubernetes_test.go

```diff
@@ -47,50 +47,161 @@ func (indexMock *IndexerMock) ByIndex(indexName, indexedValue string) ([]interfa
 	return args.Get(0).([]interface{}), args.Error(1)
 }
```

Comment:
I see there is a `kubernetes-mock.go` file. Would it make sense to move the mocks defined in this file to that file?